static/src/views/Chat.vue

```diff
@@ -69,15 +67,13 @@ export default {
         ROOMS_EMPTY: 'Add your configured bot to a Telegram group and refresh to get started',
         ROOM_EMPTY: 'No group or individual chat selected',
       },
-      messageActions: [
-        {
+      messageActions: [{
           name: 'flagMessage',
           title: 'Flag Message',
-        },
-        {
+        }, {
```

Comment:
totally subjective, but i actually prefer the original layout. :man_shrugging:

Reply:
This is before I blew everything up again. I was trying to get under 350 LoC (pointless).